### DISCUSSION TITLE
Read Image Meta Data in pre processing stage

### DISCUSSION THREAD

**Discussion by ganapathy1** [dita-ot/dita-ot#4521] [Q&A]
Read Image Meta Data in pre processing stage:
I have used "org.dita.dost.module.ImageMetadataModule". This module converted only for image format extension kind of files (like jpg,png,gig...). The same not metadata converted for high resolution PDF file. Please suggest and if you have any ideas.

**Comment by jelovirt** [comment]
@ganapathy1 When you say "metadata converted for high resolution PDF file", do you mean you're using PDF files as images?
